listOfOSBCs.md

```diff
@@ -13,7 +13,12 @@ The idea is to have a list of communities and write a short paragraph about them
 - [EMBOSS](http://emboss.sourceforge.net/what/)
 - [BioMoby](http://www.biomoby.org/)
 - [GMOD](http://gmod.org/wiki/Main_Page)
+- [mothur](http://www.mothur.org/)
+- [Rosalind](http://rosalind.info/)
+- [Open Babel](http://openbabel.org/)
```

Comment:
Awesome! I was not sure if we want to expand to Cheminformatics, if we do we should add RDKit as well :)